README.md

```diff
@@ -46,6 +46,12 @@ $ docker-compose up
 $ npm install
 ```
 
+## Initialize submodules
+```bash
+$ git submodule init
+$ git submodule update
```

Comment:
should this be git submodule update --init --recursive?

Reply:
Yea you're right that's probably better